packager/mpd/base/adaptation_set.cc

```diff
@@ -288,6 +288,13 @@ std::optional<xml::XmlNode> AdaptationSet::GetXml() {
       return std::nullopt;
     }
   }
+  if (subsegment_start_with_sap_ == 1) {
```

Comment:
Why store a uint8_t, then only use it if it's 1?  Other values are defined in the spec, such as 2.  Could this maybe set the integer attribute with any non-zero value?

Reply:
I made this change and updated the PR, and then approved it.  If you agree with the change, feel free to merge.  Otherwise, you can undo this part.